kinoml/modeling/homology.py

```diff
@@ -0,0 +1,272 @@
+from ..core.proteins import ProteinStructure
+from ..core.sequences import KinaseDomainAminoAcidSequence
+
+
+class HomologyModel:  #  TODO inherent a Base class?
+    def __init__(self, *args, **kwargs):
+        from appdirs import user_cache_dir
+
+        self.alipath = f"{user_cache_dir()}/alignment.ali"
+        #  TODO specify id, template, and sequence here? e.g.:
+
+        #  self.identifier = identifier
+        #  self.template = template
+        #  self.sequence = sequence
+
+    def get_pdb_template(
```

Comment:
This function would be useful outside of this class. Maybe even as part of the `Structure` class.

Reply:
Agreed. At the moment there is an issue with the BLAST search timing out. I'm not sure of a clear way around this, it does impeded functionality somewhat as it appears to be (somewhat) random. See [here](http://prody.csb.pitt.edu/tutorials/structure_analysis/blastpdb.html).